internal/controller/filesystem_helpers.go

```diff
@@ -227,9 +232,14 @@ func newLvmBlockDevice(ctx context.Context, c client.Client, nnfNodeStorage *nnf
 		return nil, dwsv1alpha2.NewResourceError("could not get logical volume name").WithError(err).WithMajor()
 	}
 
+	percentVG := 100
+	if nnfNodeStorage.Spec.SharedAllocation {
+		percentVG = 100 / nnfNodeStorage.Spec.Count
```

Comment:
```suggestion
		percentVG = percentVG / nnfNodeStorage.Spec.Count
```

Reply:
I guess that suggestion doesn't make sense to me.  Those two '100's are not the same thing.

Reply:
Yeah, the `percentVG := 100` is setting the value of "$PERCENT_VG" to be 100% in the case where the VG is not shared and there's only a single LV. The `percentVG = 100 / nnfNodeStorage.Spec.Count` calculation is for when the VG is shared and we need to divide the VG into equal sized chunks.